microshift_networking/microshift-configuring-routes.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-configuring-routes"]
+= Configuring routes
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-configuring-routes
+
+toc::[]
+
+You can configure routes for {microshift-short} for clusters.
+
+//OCP module, edit with care; Creating an insecure/http route
+include::modules/nw-creating-a-route.adoc[leveloffset=+1]
+
+//OCP module, edit with care; HTTP Strict Transport Security
+include::modules/nw-enabling-hsts.adoc[leveloffset=+2]
+
+//OCP module, edit with care; Enabling HTTP strict transport security per-route
+include::modules/nw-enabling-hsts-per-route.adoc[leveloffset=+2]
+
+//OCP module, edit with care; Disabling HTTP strict transport security per-route
+include::modules/nw-disabling-hsts.adoc[leveloffset=+2]
+
+//Enforcing HTTP strict transport security per-domain
+include::modules/microshift-nw-enforcing-hsts-per-domain.adoc[leveloffset=+2]
+
+//OCP module, edit with care; Troubleshooting Throughput Issues
+include::modules/nw-throughput-troubleshoot.adoc[leveloffset=+1]
+
+//OCP module, edit with care; Using cookies to keep route statefulness
+include::modules/nw-using-cookies-keep-route-statefulness.adoc[leveloffset=+1]
+
+//OCP module, edit with care; Using cookies to keep route statefulness
+include::modules/nw-annotating-a-route-with-a-cookie-name.adoc[leveloffset=+2]
+
+//OCP module, edit with care
+include::modules/nw-path-based-routes.adoc[leveloffset=+1]
+
+//OCP module, edit with care; just use the `Route` CR
+include::modules/nw-http-header-configuration.adoc[leveloffset=+1]
+
+//OCP module, edit with care
+include::modules/nw-route-set-or-delete-http-headers.adoc[leveloffset=+1]
+
+//OCP module, edit with care
+include::modules/nw-ingress-creating-a-route-via-an-ingress.adoc[leveloffset=+1]
+
+//OCP module, edit with care
+include::modules/nw-ingress-edge-route-default-certificate.adoc[leveloffset=+1]
+
+//OCP module, edit with care
+include::modules/nw-ingress-reencrypt-route-custom-cert.adoc[leveloffset=+1]
+
+[id="microshift-secured-routes_{context}"]
+== Secured routes
+
+Secure routes provide the ability to use several types of TLS termination to serve certificates to the client. The following links to the {OCP} documentation describe how to create re-encrypt, edge, and passthrough routes with custom certificates.
+
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-version}/html/networking/configuring-routes#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes[Creating a re-encrypt route with a custom certificate]
```

Comment:
Link 404s, maybe because of the {ocp-version}?

Reply:
Yeah, I can double check it once we release.